packages/table/src/cell/formats/jsonFormat.tsx

```diff
@@ -30,26 +29,25 @@ export interface IJSONFormatProps extends ITruncatedFormatProps {
     stringify?: (obj: any) => string;
 }
 
-@PureRender
```

Comment:
My understanding is that because column resize doesn't directly result in props or state changes, a `@PureRender` `JSONFormat` component will never re-render on resize. Because `JSONFormat` renders a `TruncatedFormat` and the detect truncation code is triggered by `TruncatedFormat#componentDidUpdate`, the detect truncation code will also not run on resize.